evap/static/ts/src/csrf-utils.ts

```diff
@@ -12,6 +12,7 @@ function getCookie(name: string): string | null {
     return null;
 }
 const csrftoken = getCookie("csrftoken")!;
+export const CSRF_HEADER = { "X-CSRFToken": csrftoken };
```

Comment:
small nit: As this is an object / a collection of headers (with one element), this should be called `CSRF_HEADERS`